gulpfile.js

```diff
@@ -17,6 +17,8 @@
             ' * ${pkg.description}\n' +
             ' * \n' +
             ' * @version ${pkg.version}\n' +
+            ' * @date ' + (new Date()).toISOString() + '\n' +
```

Comment:
`@date` and `@environment` don't appear in the [jsdoc list of tags ](http://usejsdoc.org/index.html). Does that matter?